src/firefox/data/scripts/firefox_browser_api.ts

```diff
@@ -50,4 +50,8 @@ class FirefoxBrowserApi implements BrowserAPI {
   public bringUproxyToFront = () => {
     port.emit('showPanel');
   }
+
+  public showNotification = (notificationText :string) => {
+    // TODO implement!
```

Comment:
Didn't we used to have notifications in Firefox?  Can you enter a new GitHub issue to reimplement notifications here?